drivers/sqlboiler-mysql/driver/mysql_test.go

```diff
@@ -18,13 +18,15 @@ import (
 	"os/exec"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+
```

Comment:
I think this blank space is redundant.

Reply:
Yes, my IDE just likes to group packages from the main module separately. Fixed.